dependencies.yaml

```diff
@@ -60,7 +60,7 @@ libevse-security:
 # OCPP
 libocpp:
   git: https://github.com/EVerest/libocpp.git
-  git_tag: 3fef09231f0033a4af5d0af97d35851ecc00d399
+  git_tag: feature/composite-schedules-api
```

Comment:
Yes so since the mentioned pull request is merged (isn't it?) you can set this to a correct version.